docs/src/rules/no-extra-boolean-cast.md

```diff
@@ -137,11 +146,23 @@ if ((foo || bar) && baz) {
     //...
 }
 
-foo && bar ? baz : bat
+var foo = new Boolean(bar || baz);
+
+foo && bar ? baz : bat;
+
+const ternaryBranches = Boolean(bar ? baz : bat);
 
-var foo = new Boolean(bar || baz)
+const nullishCoalescingOperator = Boolean(bar ?? baz);
+
+const commaOperator = Boolean((bar, baz, bat));
+
+// another comma operator example
+for (let i = 0; console.log(i), i < 10; i++) {
+    // ...
+}
 
-var foo = !!bar || baz;
```

Comment:
Looks like this example is removed, can we leave it?

Reply:
Good point. I added a few comments with it, too, since it's the only one that's not the result of a fix from the `incorrect` tab (which is probably why I forgot to include it when moving things around), and the reason that it's `correct` is not obvious at first glance